src/transform/src/cse/relation_cse.rs

```diff
@@ -17,30 +17,26 @@
 
 use std::collections::HashMap;
 
-use crate::inline_let::InlineLet;
 use mz_expr::visit::VisitChildren;
 use mz_expr::{Id, LocalId, MirRelationExpr, RECURSION_LIMIT};
 use mz_ore::stack::{CheckedRecursion, RecursionGuard};
 
-use crate::update_let::UpdateLet;
+use crate::normalize_lets::NormalizeLets;
```

Comment:
There's a comment above this it might be helpful to update, now that InlineLet/UpdateLet no longer exist.

Reply:
Thanks! Fixed, and a few other vestigial references to `UpdateLet` and `InlineLet`.